packages/web-components/src/components/ic-classification-banner/ic-classification-banner.tsx

```diff
@@ -57,9 +67,9 @@ export class ClassificationBanner {
             {classification === "default"
               ? classificationText[classification]
               : `${upTo ? "up to" : ""} 
-               ${country !== "" ? country : ""} 
```

Comment:
@GCHQ-Developer-530 as you can see here, if country is an empty string, it will render an empty string anyways